src/doc/unstable-book/src/compiler-flags/profile.md

```diff
@@ -0,0 +1,21 @@
+# `profile`
+
+The tracking issue for this feature is: None
+
+------------------------
+
+This feature allows the generation of code coverage reports.
+
+Set the compiler flags `-Ccodegen-units=1 -Clink-dead-code -Cpasses=insert-gcov-profiling -Zno-landing-pads` to enable gcov profiling.
```

Comment:
Shouldn't `-Z profile` be mentioned here? If all these other flags are required, could `-Z profile` imply them?